Subject: Packit: initial enablement
From: Lokesh Mandvekar lsm5@fedoraproject.org Mon Dec 5 17:14:30 2022 +0530
Date: Thu Jan 19 14:04:42 2023 -0500:
Git: a63b40aae3835a8b82b23755f1ed45e526af80f9

This commit enables Packit `copr_build` tasks which will run on every PR
and build RPMS using the spec file present upstream with Source0 as
the archive created from HEAD commit of the PR.

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

diff --git a/.packit.yaml b/.packit.yaml
new file mode 100644
index 00000000..15798fe9
--- /dev/null
+++ b/.packit.yaml
@@ -0,0 +1,19 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+upstream_package_name: virt-manager
+downstream_package_name: virt-manager
+
+specfile_path: virt-manager.spec
+
+jobs:
+  - job: copr_build
+    # Run on every PR
+    trigger: pull_request
+    # Defaults to x86_64 unless architecture is explicitly specified
+    targets:
+      - fedora-rawhide-aarch64
+      - fedora-rawhide-i386
+      - fedora-rawhide-ppc64le
+      - fedora-rawhide-s390x
+      - fedora-rawhide-x86_64
